indico/web/flask/util.py

```diff
@@ -240,7 +240,7 @@ def _is_office_mimetype(mimetype):
 
 
 def send_file(name, path_or_fd, mimetype, last_modified=None, no_cache=True, inline=None, conditional=False, safe=True,
```

Comment:
I think we always pass those args as kwargs, so we could finally make everything else keyword-only. Needs another look over the existing `send_file()` calls though to confirm.

```suggestion
def send_file(name, path_or_fd, mimetype, *, last_modified=None, no_cache=True, inline=None, conditional=False, safe=True,
```